src/main/java/org/aludratest/junit/AludraTestJUnitSuite.java

```diff
@@ -77,7 +77,7 @@ public AludraTestJUnitSuite(Class<?> testClass) throws InitializationError {
         String suiteName = System.getProperty(SUITE_SYSPROP);
 
         if (StringUtil.isEmpty(suiteName)) {
-            LOGGER.debug("SuiteName:\"" + suiteName + "\"");
+            LOGGER.debug("SuiteName:\"{}\"", suiteName);
```

Comment:
Does this make sense at all? Suite name is either null or an empty string. Why logging it at all here?